ersilia/hub/fetch/fetch.py

```diff
@@ -233,10 +236,14 @@ def _fetch(self, model_id):
         self.logger.debug("Fetching in your system, not from DockerHub")
         self._fetch_not_from_dockerhub(model_id=model_id)
 
-    def fetch(self, model_id):
-        self._fetch(model_id)
+    async def fetch(self, model_id):
+        await self._fetch(model_id)  
+        self._standard_csv_example(model_id)
```

Comment:
This was removed in [here](https://github.com/ersilia-os/ersilia/pull/1311/files#diff-da367f6acda96f79484d9ac4e6bf8a92230b0705f7f34827616a7b7bef6d20faR216) but it is important for Ersilia's functioning and is therefore being reintroduced again.